contracts/MemoworldNFTMint01.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.6 <0.9.0;
+pragma abicoder v2;
+
+import {IMemoworldNFT} from './interfaces/IMemoworldNFT.sol';
+
+contract MemoworldNFTMint01 {
+  // Address representating ETH (e.g. in payment routing paths)
+  address public constant ETH = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+
+  // Indicates that this plugin requires delegate call
+  bool public immutable delegate = true;
+
+  event RouteMint(address indexed sender, uint256 indexed tokenId, uint256 indexed amount, string data);
+
+  constructor() {}
+
+  function execute(
+    address[] calldata path,
+    uint256[] calldata amounts,
+    address[] calldata addresses,
+    string[] calldata data
+  ) external payable returns (bool) {
+    IMemoworldNFT memoContract = IMemoworldNFT(addresses[addresses.length - 1]);
```

Comment:
The address here needs to be hardcoded and `MemoworldNFT` needs to be deployed prior to this plugin. It also needs to be part of this review.

Reply:
Thank you for your review. OK, I will modify address to hardcoded later.
MemoworldNFT contract is on testnet (mumbai) now.  Here it is: https://mumbai.polygonscan.com/address/0xD71bD4F16391be5e108E08f2C54eac458bB9c8f7

I will deploy it to mainnet and update the address before deploy the plugin.